### ISSUE TITLE
Youtube and vimeo no cookies

### ISSUE THREAD

**Issue by DorienD** [driebit/ginger#624]
Youtube:
https://www.youtube-nocookie.com/embed/6nRQH_gpxzw

Vimeo:
?dnt=1
https://vimeo.zendesk.com/hc/en-us/articles/360001494447-Using-Player-Parameters

**Comment by mworrell** [comment]
Just a small note: the 'youtube-nocookie' site doesn't place cookies, but does place an unique `yt-remote-device-id` in the local storage. So it is not free of trackers.

**Comment by DorienD** [comment]
Hmmm so when a visitor would deny all other cookies than the functional ones, youtube should still be hidden @mworrell

**Comment by mworrell** [comment]
What we (in LearningStone) now do is to show the static preview and replace it with the embed after it is clicked.

In this way the page loads faster and we don't have cookies on loading our own page.

**Comment by ddeboer** [comment]
Yep, this is a good and common solution to the cookie/localstorage problem.
